packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -54,6 +70,18 @@ export const setQueryTaggingDebounce = wireCommit('setQueryTaggingDebounce');
  */
 export const setSessionDuration = wireCommit('setSessionDuration');
 
+/**
+ * Tracks the tagging of the query using a debounce which ends if the user
+ * accepts a query.
+ *
+ * @public
+ */
+export const trackWire = moduleDebounce(
+  wireDispatch('track'),
+  ({ state }) => state.config.queryTaggingDebounceMs,
+  'SearchTaggingChanged'
```

Comment:
What are you trying to do here? this wire is used in the `SearchTaggingChanged`, and it is racing also against the `SearchTaggingChanged` event? Racing means that when that event is emitted, the wire won't be executed.

The tagging is still failing when you are fast enough to clear the query:
![image](https://user-images.githubusercontent.com/68222542/142892185-7997d277-ecf4-4a26-8c17-938e336dba0d.png)

Reply:
I misunderstood you before sorry.

With the `UserIsTypingAQuery` event the error is gone but I'm not sure if it's the best event.